msprime/cli.py

```diff
@@ -187,34 +187,25 @@ def print_trees(self, tree_sequence, output):
         same tree. Therefore, we must keep track of all breakpoints from the
         simulation and write out a tree for each one.
         """
-        # breakpoints = self._simulator.breakpoints + [self._num_loci]
+        breakpoints = self._simulator.breakpoints + [self._num_loci]
         if self._num_loci == 1:
             tree = next(tree_sequence.trees())
             newick = tree.newick(precision=self._precision)
             print(newick, file=output)
         else:
-            # j = 0
```

Comment:
This needs cleaning up.

Reply:
I don't see the issue here?

Reply:
We want to get rid of the commented out code in this block and restore it to it's original state.

Reply:
I see. But this is the part where the bug for the mspms output with breakpoints appeared. So going back to the original state would reintroduce this bug. I tried to be identical to the new version where this bug has been corrected.

Reply:
Ah, I see, sorry I got confused about where this PR was going. looks like I need to merge this and then rebase my PR against upstream.

Reply:
Ok, so I will rebase this pull request stuff first and then we should be ready to merge this, correct?